queries.sql

```diff
@@ -1,3 +1,44 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/* Select animals ends with 'mon'*/
+SELECT * from animals
+WHERE name LIKE '%mon'
```

Comment:
- [OPTIONAL] It is a good practice to always use a statement terminator at the end of an  SQL command. Take a look at this [explanation](https://www.mssqltips.com/sqlservertip/6593/sql-semicolon-required-usage/#:~:text=The%20semicolon%20(%3B)%20is%20used,statements%20it%20is%20not%20mandatory.&text=I%20personally%20use%20semicolons%20only,going%20away%22%20are%20quite%20uncertain.)

Reply:
Oh, thank you for this tip.  I will take care of it in the future.